xbmc/utils/SystemInfo.cpp

```diff
@@ -867,6 +868,40 @@ bool CSysInfo::HasHW3DInterlaced()
   return false;
 }
 
+bool CSysInfo::HWSupportsStereo(const int mode)
+{
+#if defined(TARGET_ANDROID)
```

Comment:
any reason to limit it for android? Can't we use HAVE_LIBAMCODEC or what its called here? (or anything else which identifies aml - we could even just probe the filesystem i guess). This might be valid for openelec for example as well whcih is not android...